ci/render_task.py

```diff
@@ -17,49 +17,58 @@
 
 def main():
     parser = argparse.ArgumentParser()
+    parser.add_argument('--use-secrets-server', action='store_true')
     parser.add_argument('--outfile', default='tasks.yaml')
+    parser.add_argument('--minimal', action='store_true', help='omit prebuild and promote steps')
 
     parsed = parser.parse_args()
+    full_build = not parsed.minimal
```

Comment:
:-/

as stated above. this will not be very sustainable. it is very imminent that we will have more than two different "build flavours"